test/Backends/Michelson.hs

```diff
@@ -602,9 +519,7 @@ identityAppTerm2 =
     $ J.LamM [] ["x"]
     $ Ann one (primTy (Untyped.pair opl Untyped.unit))
     $ J.AppM
-      ( Ann
-          one
-          (J.Pi one primPairTy (primTy (Untyped.pair opl Untyped.unit)))
+      ( Ann one (J.Pi one primPairTy (primTy (Untyped.pair opl Untyped.unit)))
```

Comment:
I prefer the old style for this, as it allowed the arguments to align with the $